### DISCUSSION TITLE
Some privacy questions

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1644] [Q&A]
Hello!
Please tell me the following:

1. Does SimpleLogin hide the headers of emails sent using your service? I mean, if the recipient of an email sent with SimpleLogin opens its headers, will my real address be hidden?
2. Does SimpleLogin hide my IP address from the headers?

I look forward to hearing from you!

**Comment by nguyenkims** [comment]
1. Yes SimpleLogin remove existing headers. Your real address won't be revealed in the header.
2. Yes.
